docker-compose-github-dbdocs-action.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  graphql-engine:
+    image: hasura/graphql-engine:v2.43.0
```

Comment:
Thinking out loud but it might be worth adding this to the launch checklist so we remember to bump the Hasura version here too

Reply:
great call—i'll update the launch checklist